packages/menu/src/MenuContext.tsx

```diff
@@ -4,10 +4,12 @@ import { Theme } from '@leafygreen-ui/lib';
 
 interface MenuData {
   theme: Theme;
+  darkMode: boolean;
 }
 
 export const MenuContext = createContext<MenuData>({
   theme: Theme.Light,
+  darkMode: false,
```

Comment:
is it dangerous to set a default here? will we overwrite a value that comes in from the LG provider?

Reply:
I don't think so, TS complains if nothing is set but it is overwritten when we pass in a value.